models/index.js

```diff
@@ -0,0 +1,30 @@
+// import models
+const User = require('./User')
+const Order = require('./Order');
+const OrderItems = require('./OrderItems');
+const Item = require('./Item');
+const Category = require('./Category');
+
+// Create associations
+
+
+// Items belongsTo Category
+Item.belongsTo(Category)
+// Categories have many Items
+Category.hasMany(Item)
+// Products belongToMany Orders (through OrderItems)
+Item.belongsToMany(Order, {
+  through: OrderItems
+})
+// Order belongToMany Items (through OrderItems)
+Order.belongsToMany(Item, {
+  through: OrderItems
+})
```

Comment:
Looked at the documentation. I think this is all we need. 

```suggestion
Category.hasMany(Item);

Item.belongsToMany(Order, {
  through: OrderItems
});

User.hasMany(Order);
```